### DISCUSSION TITLE
How to log to console at execution but avoid logging other tests

### DISCUSSION THREAD

**Discussion by Formartha** [pytest-dev/pytest#13074] [General]
Hi,
As we use pytest for e2e tests with logs of logs, we decided to use log_cli=false  on our CI.
However, on conftest session start we are logging useful information of the test session (mostly environment discovery items), but when using log_cli=false  the test session avoids logging it.

What are our options?

Thanks!

**Comment by RonnyPfannschmidt** [comment]
Pytest-metadata is a plug-in that's helping 

Also the terminal header hooks

**Reply by Formartha** [reply]
Thanks, will try and report back

**Reply by Formartha** [reply]
Okay I tested it, it won't help as in the session start there are logic of things we are making that produces the data we're expecting to see in the terminal.

**Comment by afcmrp** [comment]
It sounds like you should be able to disable the StreamHandler for all loggers and only enable it for the specific logger you create in conftest?
